src/main/java/com/powsybl/openloadflow/ac/outerloop/AreaInterchangeControlOuterloop.java

```diff
@@ -61,6 +66,7 @@ public void initialize(AcOuterLoopContext context) {
         var contextData = new DistributedSlackContextData();
         context.setData(contextData);
         LfNetwork network = context.getNetwork();
+        listBusesNoArea(network);
```

Comment:
Rename to listBuses**W**oArea ?

Reply:
I prefer this too, renamed in dae8119

Reply:
fixed in 78188368bd470c602002e0284023d932d6625f9c